packages/sdk/react-shell/src/panels/IdentityPanel/steps/AgentPanel.tsx

```diff
@@ -0,0 +1,161 @@
+//
+// Copyright 2023 DXOS.org
+//
+
+import React, { useState, useCallback, useEffect } from 'react';
+import { type Event, type SingleOrArray } from 'xstate';
+
+import { useClient, useAgentHostingProviderClient } from '@dxos/react-client';
+import { type Identity, useHaloInvitations } from '@dxos/react-client/halo';
+import { Invitation, InvitationEncoder } from '@dxos/react-client/invitations';
+import { useTranslation } from '@dxos/react-ui';
+
+import { Action, Actions, StepHeading, Input } from '../../../components';
+import { type IdentityPanelStepProps } from '../IdentityPanelProps';
+import { type IdentityEvent } from '../identityMachine';
+
+export interface AgentFormProps extends Omit<IdentityPanelStepProps, 'send'> {
```

Comment:
The general pattern is to use `type` defs with the Props suffix associated with a component. I don't see the need for the two defs?